core/extension/gdextension.cpp

```diff
@@ -35,6 +35,7 @@
 #include "core/object/method_bind.h"
 #include "core/os/os.h"
 #include "core/version.h"
+#include "gdextension.compat.inc"
```

Comment:
Should be moved above, making it:
```cpp
#include "gdextension.h"
#include "gdextension.compat.inc"

#include "core/config/project_settings.h"
```

Reply:
When I tried doing this I got an error from style checks:
https://github.com/godotengine/godot/actions/runs/8935217873/job/24543349469

Do you know what I did wrong?

Reply:
You missed the empty line, the header should always be separate

Reply:
aha, I didn't know the empty line mattered